Modules/core/View/Coreconnection/indexAction.php

```diff
@@ -76,13 +76,13 @@
 
                             <form class="form-signin" action="corelogin" method="post">
                                 <input name="redirection" type="hidden" value="<?php echo $redirection ?>">
-                                <input name="login" autocomplete="username" type="text" class="form-control" placeholder="<?php echo CoreTranslator::Login($language) ?>" required autofocus>
-                                <input name="pwd" autocomplete="current-password" type="password" class="form-control" placeholder="<?php echo CoreTranslator::Password($language) ?>" required>
+                                <input id="login" name="login" autocomplete="username" type="text" class="form-control log" placeholder="<?php echo CoreTranslator::Login($language) ?>" required autofocus>
```

Comment:
why log and pass class here?

Reply:
ah, maybe that's a mix between 2 pushes. I'm gonna remove classes modifications